client/src/components/Home/index.tsx

```diff
@@ -213,6 +257,42 @@ export default class Home extends React.Component<any, IHomeState> {
     return (
       <div>
         <h1>Bot Analytics</h1>
+      <div style={{textAlign:"right"}}>
+      <Button
+        tooltipLabel="Import dashboard"
+        onClick={this.onOpenImport.bind(this)}
+        component={Link}
```

Comment:
Are you sure you need the Link component?
As far as I know, you only need the button component from react md